nixos/modules/services/misc/nzbget.nix

```diff
@@ -40,35 +47,33 @@ in {
         p7zip
       ];
       preStart = ''
-        datadir=/var/lib/nzbget
         cfgtemplate=${cfg.package}/share/nzbget/nzbget.conf
-        test -d $datadir || {
-          echo "Creating nzbget data directory in $datadir"
-          mkdir -p $datadir
-        }
-        test -f $configfile || {
-          echo "nzbget.conf not found. Copying default config $cfgtemplate to $configfile"
-          cp $cfgtemplate $configfile
-          echo "Setting $configfile permissions to 0700 (needs to be written and contains plaintext credentials)"
-          chmod 0700 $configfile
+        if [ ! -d ${dataDir} ]; then
+          echo "Creating nzbget data directory in ${dataDir}"
+          mkdir -p ${dataDir}
```

Comment:
The creation of the statedirectory should be offloaded to systemds `StateDirectory=`